ui/views_test.py

```diff
@@ -330,19 +343,23 @@ def test_collection_viewset_create_as_superuser(logged_in_apiclient):
     other_user = UserFactory()
     input_data = {
         'owner': other_user.id,
+        'view_lists': [],
+        'admin_lists': [],
         'title': 'foo title'
     }
     assert client.post(url, input_data, format='json').status_code == status.HTTP_201_CREATED
 
     # if does not specify the owner id it gets a different error
     input_data = {
-        'title': 'foo title'
+        'title': 'foo title',
+        'view_lists': [],
+        'admin_lists': []
```

Comment:
Previous: a user doesn't have permission to create a collection belonging to someone else or nobody at all - so the permission issue raises an exception first.  This one: a superuser can create a collection belonging to anyone, so no permission error is raised, which eventually leads to the next error because owner_id is required in the database.